### PR TITLE
Scrolling

### PR DESCRIPTION
**Pull request by andrej-sajenko**
rebase and Clean up of PR #111

Do you need a extra PaginatedStoreCursor Interface?

I think we can move its method to StoreCursor and remove the interface completely.

### CODE REVIEW

**Comment by andrej-sajenko** [comment]
@michel-kraemer review this, please

**Comment by michel-kraemer** [comment]
'Pagination' should be called 'scrolling'. Andrej will rename the class and the parameters.

**Comment by michel-kraemer** [comment]
@andrej-sajenko is this PR ready to be reviewed?

**Comment by andrej-sajenko** [comment]
+1

**Comment by andrej-sajenko** [comment]
please review #127 first, if you approve i will merge this pull request myself

**Comment by andrej-sajenko** [comment]
review please

File: georocket-server/src/main/java/io/georocket/storage/indexed/IndexedStoreCursor.java [hunk 1]
```diff
@@ -71,6 +71,12 @@
   private ChunkMeta[] metas;
   
   /**
+   * Number of total items retrieved
```
Comment (michel-kraemer) [inline georocket-server/src/main/java/io/georocket/storage/indexed/IndexedStoreCursor.java]: Please improve comment

File: georocket-server/src/main/java/io/georocket/storage/indexed/IndexedStoreCursor.java [hunk 2]
```diff
@@ -71,6 +71,12 @@
...
+   */
+  private int currentHitCount;
+  private int currentHitSize;
```
Comment (michel-kraemer) [inline georocket-server/src/main/java/io/georocket/storage/indexed/IndexedStoreCursor.java]: Please add javadoc

File: georocket-server/src/test/java/io/georocket/http/StoreEndpointTest.java [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.RunTestOnContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import io.vertx.ext.web.Router;
+import io.vertx.rxjava.core.Vertx;
+import io.georocket.http.mocks.MockIndexer;
+import rx.Observable;
+
+@RunWith(VertxUnitRunner.class)
+public class StoreEndpointTest {
```
Comment (michel-kraemer) [inline georocket-server/src/test/java/io/georocket/http/StoreEndpointTest.java]: Javadoc

**Comment by andrej-sajenko** [comment]
@michel-kraemer as we discussed ...
* i created a FrameCursor and reused it in the IndexedStoreCursor implementation
* the elasticsearch client got a new argument for es parameters

Note: we should take a look an all Cursors and Stores (Indexed and Metadata)

review, pls

**Pull request merged**